cmd/juju/commands/bootstrap_test.go

```diff
@@ -590,6 +591,154 @@ func (s *BootstrapSuite) TestBootstrapDefaultConfigStripsInheritedAttributes(c *
 	c.Assert(ok, jc.IsFalse)
 }
 
+// checkConfigs runs bootstrapCmd.getBootstrapConfigs and checks the returned configs match
+// the expected values passed in the expect parameter.
+func checkConfigs(
+	c *gc.C,
+	bootstrapCmd bootstrapCommand,
+	key string,
+	ctx *cmd.Context, cloud *cloud.Cloud, provider environs.EnvironProvider,
```

Comment:
+1. Prefer long functions to be made to look like structs. Inconsistent numbers of parameters on a line is frowned upon.